code/modules/events/tear.dm

```diff
@@ -29,6 +33,10 @@
 		for(var/obj/machinery/M in range(8, T))
 			INVOKE_ASYNC(M, TYPE_PROC_REF(/atom, get_spooked))
 
+		return
+
+	log_and_message_admins("dimensional tear failed to find a valid turf in [impact_area]")
```

Comment:
most events make use of log_debug, could be good to include such a call here

Reply:
Agreed, also `log_and_message_admins` uses keyname usr, which is probably going to break here

Reply:
switched to log_debug here!